src/acsets/LVectors.jl

```diff
@@ -0,0 +1,40 @@
+module LVectors
+export LVector
+
+using CompTime
+
+struct LVector{T,Labels} <: AbstractVector{T}
```

Comment:
Perhaps add a docstring explaining that this type provides similar functionality to the corresponding type in LabeledArrays.jl but without all the dependencies.